resources/webviews/common/detector.html

```diff
@@ -30,6 +30,28 @@ <h2 class="content-header">{{name}}</h2>
         </div>
         <hr/>
             {{#eachProperty detectorData}}
+            {{#if (hasResourceThrottle value)}}
```

Comment:
Could we use an `ifEquals` helper here (like [this](https://stackoverflow.com/a/34252942))?

That way we won't need to keep adding additional helpers for similar string equality checks in future?